tests/storage/restricted_namespace_cloning/conftest.py

```diff
@@ -252,3 +287,18 @@ def vm_for_restricted_namespace_cloning_test(
     ) as vm:
         running_vm(vm=vm, wait_for_interfaces=False)
         yield vm
+
+
+@pytest.fixture()
+def user_has_get_permissions_in_source_namespace(
+    namespace, unprivileged_client, data_volume_multi_storage_scope_module
+):
+    dv_name = data_volume_multi_storage_scope_module.name
+    assert list(DataVolume.get(namespace=namespace.name, name=dv_name, dyn_client=unprivileged_client)), (
```

Comment:
if you already know the name of the dv, then you don't need get() call like this. Using python object by name and namespace and doing exists, should be good enough